manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: danielchalmers.SteamAccountSwitcher
+PackageVersion: 2.3.1

```

```suggestion
PackageVersion: 2.3.1.0
```
This must match Control Panel.